apps/pollbook/frontend/src/error_screen.tsx

```diff
@@ -16,6 +22,7 @@ export function ErrorScreen(): JSX.Element {
           <Icons.Danger />
         </FullScreenIconWrapper>
         <H1>Something went wrong</H1>
+        <P>{ERROR_SCREEN_MESSAGES.RESTART}</P>
```

Comment:
Didn't add the note to reach out to VotingWorks to VxPollBook since it's not bound by the same certification process and we personally didn't want to have to add this additional copy to these screens